### ISSUE TITLE
Create NPB disk image

### ISSUE THREAD

**Issue by powerjg** [darchr/gem5#112]
And get these workloads running. 

 - [ ] Document what's needed to do this.
 - [ ] Figure out kernel version from Pouya
 - [ ] Create disk image

- [ ] A size
- [ ] B size 
- [ ] C size

**Comment by ayaz91** [comment]
Did some tests with an NPB disk image for class S workloads with O3 cpu. The workloads execute for the most part, but crash at the end of the execution. Still looking into the problem

**Comment by powerjg** [comment]
Can you show us the error?

**Comment by ayaz91** [comment]
This is an example of the error (for ep benchmark): 

At line 273 of file ep.f (unit = 6, file = 'stdout')
Fortran runtime error: Unexpected element '('EP Benchmark Results:'//'CPU Time =',f10.4/'N = 2^',

**Comment by powerjg** [comment]
That doesn't look like a gem5 bug.

It looks like a weird compiler + runtime issue.

The error is on this line: https://github.com/darchr/npb/blob/master/NPB3.3-OMP/EP/ep.f#L274.

This works fine when running in 100% KVM mode? That seems surprising to me.  

I guess it's possible there's a weird memory bug in the simulator where the runtime is getting a value that it doesn't expect even though the correct value is in the program...

**Comment by ayaz91** [comment]
Yeah, this works fine when run in 100% KVM mode. Also, when I use Pouya's disk image with my kernel (4.19), this benchmark seems to work fine with O3

**Comment by powerjg** [comment]
Wow. That's super weird!

The problem seems to be with printing something, so it could be a problem with the I/O drivers or devices.

Have you tried your disk image with Pouya's kernel? Or, I wonder if it works in Ruby? Could it be the problem with X86+classic caches.

**Comment by powerjg** [comment]
What's the state of this issue?
